rad_common.gemspec

```diff
@@ -82,4 +82,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'vcr'
   s.add_development_dependency 'webmock'
   s.add_development_dependency 'wkhtmltopdf-binary'
+  s.add_development_dependency 'tty-prompt'
```

Comment:
@calebtocco when you get to this, do you remember how this works, if I'm adding a dev dependency does that mean that we will need to add it to the Gem file for each project?